Mathlib/Logic/Equiv/LocalEquiv.lean

```diff
@@ -827,19 +827,19 @@ theorem eqOnSource_refl : e ≈ e :=
   Setoid.refl _
 #align local_equiv.eq_on_source_refl LocalEquiv.eqOnSource_refl
 
-/-- Two equivalent local equivs have the same source -/
+/-- Two equivalent local equivs have the same source. -/
```

Comment:
Does the style guide have anything to say about periods in docstrings? If not, I think we should just leave things as is (at least, until someone changes the style guide).

Reply:
I checked this :-) The [doc conventions](https://leanprover-community.github.io/contribute/doc.html#doc-strings) mention this: "If a doc string is a complete sentence, then it should end in a period."

The style guide doesn't mention this - one could argue that such a rule better belongs into the doc conventions anyway.